pkgs/applications/emulators/dolphin-emu/default.nix

```diff
@@ -57,21 +57,17 @@
 
 stdenv.mkDerivation rec {
   pname = "dolphin-emu";
-  version = "5.0-19870";
+  version = "5.0-20347";
 
   src = fetchFromGitHub {
     owner = "dolphin-emu";
     repo = "dolphin";
-    rev = "032c77b462a220016f23c5079e71bb23e0ad2adf";
-    sha256 = "sha256-TgRattksYsMGcbfu4T5mCFO9BkkHRX0NswFxGwZWjEw=";
+    rev = "dc0814ae4622313d513468bdc377ee9c031de199";
```

Comment:
how do you match a version to a rev?

Reply:
`nix-shell maintainers/scripts/update.nix --argstr package dolphin-emu` or check https://dolphin-emu.org/download/